SFDigitalServices/sf-dahlia-web
# Remove whiteout class from listing details subheaders #173712407

Merging blocked by pulling in pattern lib changes from. https://github.com/Exygy/sf-dahlia-pattern-library/pull/130

Ticket: https://www.pivotaltracker.com/story/show/173712407
Review instructions:
1. Visit a sample rental listing, make sure you can see "household size", "minimum", and "maximum" subheaders for each category. make sure "RENT" subheader is still hidden
1. Check that subheaders are also visible for sale listings
